Sources/Fullscreen/MessageForm/MessageFormView.swift

```diff
@@ -41,6 +41,17 @@ class MessageFormView: UIView {
         }
     }
 
+    var inputEnabled: Bool = true {
+        didSet {
+            textView.isUserInteractionEnabled = inputEnabled
+            if inputEnabled {
+                _ = becomeFirstResponder()
+            } else {
+                _ = resignFirstResponder()
+            }
```

Comment:
Just to get rid of one of the `_`s:

```swift
inputEnabled ? becomeFirstResponder() : resignFirstResponder()
```

Or do you get a complaint from Xcode/SwiftLint if you drop it?

Reply:
Seems to go through without warning

Reply:
Awesome 👍
Otherwise, just add `@discardableResult` to both methods.

Reply:
You should probably add the annotation anyways since they're public methods. I don't think we usually care about the result.

Reply:
Ya, kind of prefer this style:
```
            if inputEnabled {
                becomeFirstResponder()
            } else {
                resignFirstResponder()
            }
```

The ternary operator implies to me that we're _first and foremost_ producing a result, but we really aren't in this particular case.